themes/poetry/layouts/partials/documentation_flyover.html

```diff
@@ -1,3 +1,8 @@
+{{ $currentPage := . }}
+{{ $docsDir := "docs/" }}
```

Comment:
A default value is required in case the header is displayed on a non-documentation page (for instance, [blog](https://python-poetry.org/blog/) or [home](https://python-poetry.org/)), where we have no version to rely on (so we default to the stable one).